src/rust/engine/process_execution/src/remote.rs

```diff
@@ -134,19 +137,24 @@ impl CommandRunner {
                                 )
                               )
                             })
-                            .map(|operation| future::Loop::Continue(operation))
+                            .map(|operation| future::Loop::Continue((operation, 0))) // iter_num does not matter for `MissingDigests`
```

Comment:
I'd maybe phrase this as "Reset iter_count on MissingDigest" rather than that it does not matter